docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,232 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+parablock must be generated and go through the entire backing process in a single relay block. It
+then proceeds to the inclusion process while generating the next relay-chain block.
+
+Asynchronous backing brings the following changes to the parachain protocol:
 
-**Unincluded segments** are chains of candidate blocks that are not yet included in the relay chain.
-Parablocks can be added to this unincluded segment without waiting for the latest included parent
-block of the relay chain. The core functionality that asynchronous backing brings is the ability to
-build on these unincluded segments of block ancestors rather than ancestors included in the relay
-chain state.
+- Parablock generation is now decoupled from the backing process, avoiding the previous 6-second
+  deadline through the backing process. This enables for more involved scaling approaches, such as
+  giving more time for collators to properly use blockspace or increasing block velocity based on
+  demand.
 
-Currently, parablocks rely on the most recent relay chain block (often referred to as the **parent**
-block, as the parablock anchors itself to it). Each parablock must be generated and go through the
-entire backing process in a single relay block. It then proceeds to the availability + inclusion
-process during the next block.
+- Parachains can generate blocks and have them placed into **unincluded segments** of parablock
+  ancestors (i.e. parablocks that are seconded but not yet backed by paravalidators and included in
+  the relay chain) rather than ancestors included in the relay chain state.
 
-:::info
+### Unincluded Segments
 
-It is important to note the following clarifications:
+Unincluded segments are chains of candidate parablocks that are seconded but still need to be backed
+and included in the relay chain. Parablocks can be added to this unincluded segment without waiting
+for the latest included parent block of the relay chain. The core functionality that asynchronous
+backing brings is the ability to build on these unincluded segments of block ancestors rather than
+ancestors included in the relay chain state.
 
-1. The parablock is not placed on the relay chain in its entirety, but rather the **paraheader**
-2. The relay chain does not access the entire state of a parachain but only the values that changed
-   during that block and the merkelized hashes of the unchanged values.
+:::info Unincluded segments happen on the collator portion
+
+**Unincluded** segments are blocks that are seconded but still need to be backed. The backing
+process occurs on the relay chain, whereas unincluded segments are moreso held in the collator.
 
 :::
 
-For more information on the validity and availability process, be sure to visit the
-[parachain protocol](../learn/learn-parachains-protocol.md) page.
+Compared to synchronous backing, contextual execution shifts from being the parablock ancestors
```

Comment:
See my earlier comment for a more complete description of what's going on here. Part of the execution context still comes from the last included ancestor.